understanding/22/accessible-authentication-no-exception.html

```diff
@@ -50,12 +50,13 @@ <h2>Intent of Accessible Authentication (No Exception)</h2>
 
         <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method to log in, access content, and undertake tasks. This criterion is the same as <a href="accessible-authentication">Accessible Authentication</a> but without the exceptions for objects and user-provided content.</p>
 
-        <p>The scenarios where the two exceptions might apply are authentication mechanisms which:</p>
+        <p>There are two scenarios where there are exceptions for <a href="accessible-authentication.html">Accessible Authentication</a> but would fail this AAA criterion. These are where authentication mechanisms display a selection of images, and:</p>
```

Comment:
i'd add, for clarify "at AA" just before the "but would fail"

Reply:
maybe stylistically, replace "These are where" (which sounds a bit odd/informal) with "These relate to" or "These concern"